python/tvm/relay/op/layers.py

```diff
@@ -0,0 +1,115 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I think it is better not to introduce a new file and move its content to frontend/common.py

Reply:
done